### ISSUE TITLE
Regarding non dynamic relations

### ISSUE THREAD

**Issue by sachin040404** [facebookresearch/PyTorch-BigGraph#255]
Hi,

Is it possible for you to provide any config example with non dynamic relations (all the examples in the code are for dynamic relations) ?

**Comment by sachin040404** [comment]
also, when i try to use the non dynamic relation by passing as parameter, it throws some runtime error but not any helpful message:

2022-04-08 16:55:12,192   [Trainer-0] Loading entity counts...
2022-04-08 16:55:12,193   [Trainer-0] Creating workers...
2022-04-08 16:55:12,271   [Trainer-0] Initializing global model...
2022-04-08 16:55:12,276   [Trainer-0] Creating GPU workers...
2022-04-08 16:55:12,287   [Trainer-0] Starting epoch 1 / 30, edge path 1 / 1, edge chunk 1 / 1
2022-04-08 16:55:12,298   [Trainer-0] Edge path: edges
2022-04-08 16:55:12,299   [Trainer-0] still in queue: 0
2022-04-08 16:55:12,299   [Trainer-0] Swapping partitioned embeddings None ( 0 , 0 )
2022-04-08 16:55:12,300   [Trainer-0] Loading partitioned embeddings from checkpoint
2022-04-08 16:55:16,131      [GPU #0] GPU subprocess 0 up and running
terminate called after throwing an instance of 'std::runtime_error'
  what():  rel > numRelations
Aborted (core dumped)


=====
I am using the following config file:
def get_torchbiggraph_config():

    config = dict(  # noqa
        # I/O data
        entity_path="entity_counts",
        edge_paths=["edges"],
        checkpoint_path="model/ba5eac47",
        # Graph structure
        entities={"all": {"num_partitions": 1}},
        relations=[
            {   
                "name": "all_edges",
                "lhs": "all",
                "rhs": "all",
                "operator": "diagonal",
            }
        ],
        dynamic_relations=True,
        # Scoring model
        dimension=433,
        global_emb=False,
        comparator="dot",
        # Training
        num_epochs=50,
        num_uniform_negs=1000,
        loss_fn="softmax",
        lr=0.1,
        batch_size=20000,
        regularization_coef=1e-3,
        num_gpus=1,
        # Evaluation during training
        eval_fraction=0.0 #0.1,  # to reproduce results, we need to use all training data
    )

    return config

and commnd:
export CUDA_VISIBLE_DEVICES=0 && torchbiggraph_train config.py -p checkpoint_path=model/model_name -p batch_size=5000 -p dimension=200 -p num_gpus=1 -p checkpoint_preservation_interval=5 -p num_epochs=30 -p eval_fraction=0.00 -p edge_paths=edges -p num_batch_negs=1000 -p num_uniform_negs=50 -p dynamic_relations=false

====
Any suggestion or example config file would be helpful, i am using 21 relations ?

**Comment by sachin040404** [comment]
@lw  Please let me know if you need any further information ?
